### PR TITLE
Added keymap folder matching username to rorschach; added readme to old keympa

### PR DESCRIPTION
**Pull request by tuesdayjohn**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/keebio/rorschach/keymaps/tuesdayjohn/keymap.c [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+};
+
+enum rorschach_keycodes {
+  COLEMAK = SAFE_RANGE,
+  QWERTY
+};
+
+//Tap Dance Declarations
+enum {
+  ADJ = 0,
```
Comment (fauxpark) [inline keyboards/keebio/rorschach/keymaps/tuesdayjohn/keymap.c]:
```suggestion
  ADJ,
```

This is not really necessary, the enum numbering always starts at 0.

File: keyboards/keebio/rorschach/keymaps/tuesdayjohn/keymap.c [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case COLEMAK:
+      if (record->event.pressed) {
+        default_layer_set(1UL << _COLEMAK);
+        layer_move (_COLEMAK);
+
+      }
+      return false;
+      break;
```
Comment (fauxpark) [inline keyboards/keebio/rorschach/keymaps/tuesdayjohn/keymap.c]: `break` after `return` is dead code.

File: keyboards/keebio/rorschach/keymaps/tuesdayjohn/rules.mk [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in 
+#   the appropriate keymap folder that will get included automatically
+#
+
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```
Comment (fauxpark) [inline keyboards/keebio/rorschach/keymaps/tuesdayjohn/rules.mk]: You can remove all of these that haven't changed from the keyboard-level `rules.mk`.

File: keyboards/keebio/rorschach/keymaps/tuesdayjohn/keymap.c (hunk 2 above)
Comment (drashna) [reply keyboards/keebio/rorschach/keymaps/tuesdayjohn/keymap.c]: Honestly, I think that removing `return false` is better than removing break. 

Some functionality breaks if you return false (such as one shots).

**Pull request merged**